extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -175,6 +175,7 @@ tags: {  # <amp-story-page>
 tags: {  # <amp-story-grid-layer>
   html_format: AMP
   tag_name: "AMP-STORY-GRID-LAYER"
+  requires_extension: "amp-story"
```

Comment:
unrelated changes?

Reply:
these are enforced via the mandatory_parent (because the parent has it) but these should have it too.

Reply:
i'll remove and put in a separate pr